lib/posthog/feature_flags.rb

```diff
@@ -137,13 +150,35 @@ def get_all_flags(distinct_id, groups = {}, person_properties = {}, group_proper
         end
       end
 
-      if fallback_to_decide && !only_evaluate_locally
```

Comment:
Node library says we keep this but python one says we remove this.... not sure what's supposed to happen here :|